containerized-data-importer/models/V1beta1StorageSpec.ts

```
@@ -0,0 +1,38 @@
+import type { V1TypedLocalObjectReference } from './V1TypedLocalObjectReference';
+import type { V1TypedObjectReference } from './V1TypedObjectReference';
+import type { V1VolumeResourceRequirements } from './V1VolumeResourceRequirements';
+
+export interface V1Beta1StorageSpec {
+  /** AccessModes contains the desired access modes the volume should have. More info: https://kubernetes.io/docs/concepts/storage/persistent-volumes#access-modes-1 */
+  accessModes?: (
+    | "ReadOnlyMany"
+    | "ReadWriteMany"
+    | "ReadWriteOnce"
+    | "ReadWriteOncePod"
+  )[];
```

ESLint

[error] Replace `⏎····|·"ReadOnlyMany"⏎····|·"ReadWriteMany"⏎····|·"ReadWriteOnce"⏎····|·"ReadWriteOncePod"⏎··` with `'ReadOnlyMany'·|·'ReadWriteMany'·|·'ReadWriteOnce'·|·'ReadWriteOncePod'`

(prettier/prettier)